person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(id, age, parent_permission: true, name: 'unknown')
+    @id = id
```

Comment:
- [ ] Kindly consider placing the getters and setter before line 2. you can easily create the getters using `attr_reader` and both getter and setter using `attr_accessor`.
here is the requirement you need to fulfill:
  - Getters for @id, @name, and @age.
  - Setters for @name and @age.
- [ ] Also in the initialize parameters, kindly consider to write default values like this  `parent_permission = true, name = 'unknown'` As I have ran your code and it is giving errors.